vlib/v/checker/tests/match_last_branch_only_casted.out

```diff
@@ -0,0 +1,7 @@
+vlib/v/checker/tests/match_last_branch_only_casted.vv:6:10: error: the type of the last expression of the first match branch was `u64`, which is not compatible with `int literal`
+    4 |         'a' { 10 }
```

Comment:
That is just wrong, the type of the last expression of the first branch is `int` or `int literal`, not `u64`. Do swap the type names in the error.

Reply:
`u64` is the type of the value in the last match branch.

Reply:
Oops. my bad.

Reply:
Should be fixed now